src/app/shared/auth-guard.service.ts

```diff
@@ -70,4 +70,17 @@ export class AuthService {
     );
   }
 
+  isAuthenticated(): Observable<boolean> {

```

Comment:
Not sure what's best but I think `checkAuthentication` or `handlerAuthGuard` better describes this.